src/Object/Api/Mastodon/InstanceV2/UserStats.php

```diff
@@ -33,10 +33,10 @@ class UserStats extends BaseDataTransferObject
 	/** @var int */
 	protected $active_month = 0;
 
-	/**
-	 * @param $active_month
-	 */
-	public function __construct($active_month)
+    /**
+     * @param int $active_month
+     */
+    public function __construct(int $active_month)
```

Comment:
```suggestion
	/**
	 * @param int $active_month
	 */
	public function __construct(int $active_month)
```

Reply:
So now PHPStorm doesn't add tabs anywhere 😅

Reply:
I'm using PHPStorm as well and normally - except for JSON files (like `composer.json`)  - the indentation check works fine

Reply:
Sigh... @nupplaphil is there a config file you can throw me because I'm at wits end with these PHP IDEs and this tabs/spaces BS at this point :(.